src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdmin.cshtml

```diff
@@ -48,32 +65,29 @@ else
 
                     <div class="form-group d-inline-flex float-right mb-0">
                         <div class="float-right">
-                            @if (Model.ContainedContentTypeDefinitions.Count() == 1)
+                            <a class="btn btn-sm btn-primary" edit-for="@Model.ListPart.ContentItem" asp-route-returnUrl="@FullRequestPath">
+                                @T["Edit {0}", @Model.ListPart.ContentItem.DisplayText]
+                            </a>
```

Comment:
Why is this change and `authorizedContentTypeDefinitions.Count() > 1` below necessary?

Reply:
I think this way makes sense. The edit button is always present, so I moved it up here. And if there is more than 1, the button will be a dropdown, and if there is less than 1, we don't show anything.

Reply:
Indeed, good.